google/cloud/internal/backoff_policy.h

```diff
@@ -134,11 +134,9 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
                 std::chrono::duration<double, std::micro>>(maximum_delay)),
         scaling_(scaling),
         current_delay_start_(initial_delay_),
-        current_delay_end_((std::min)(
-            std::chrono::duration_cast<
-                std::chrono::duration<double, std::micro>>(scaling_ *
-                                                           initial_delay_),
-            maximum_delay_)) {
+        current_delay_end_(std::chrono::duration_cast<
```

Comment:
we don't need the `duration_cast`, right?

Reply:
Good catch. Done

Reply:
I don't think you need `duration_cast` on any of those initializations now.

Reply:
Done